maplexer.l

```diff
@@ -1,753 +0,0 @@
-%{
```

Comment:
There should be no other change to maplexer.l than adding the idw keyword. Also please do not add the changes to maplexer.c to the pull request